builtin/game/privileges.lua

```diff
@@ -34,15 +34,15 @@ core.register_privilege("basic_privs", "Can modify 'shout' and 'interact' privil
 core.register_privilege("privs", "Can modify privileges")
 
 core.register_privilege("teleport", {
-	description = "Can use /teleport command",
+	description = "Can teleport using /teleport",
```

Comment:
The chat command allows teleporting other players, so i think better unchanged.

Reply:
This is needed to make the difference between `bring` and `teleport` clear. `teleport` priv alone does only *partially* allow you to use `/teleport`.

Reply:
The original still wasn't wrong and simply adding the word teleport a second time doesn't explain this difference.